handbook/engineering/observability/monitoring.md

```diff
@@ -221,6 +225,27 @@ kubectl port-forward svc/prometheus 9090:30090 -n prod
 ./dev/grafana.sh
 ```
 
+### Creating Cloud only Grafana dashboards
+
+While all dashboards required to troubleshoot our product should be shipped to customers, our Cloud deployment might require additional dashboards to the ones we ship to customers, for example:
+- When the additional dashboard is not ready yet to graduate to customers
+- When the additional dashboard applies only to our Cloud deployment
+
+Dashboards can be deployed to our Cloud deployment by adding them in `json` format to `dashboards/files` in https://github.com/sourcegraph/deploy-sourcegraph-dot-com/. To learn more, reference its [documentation](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/tree/feature/release/dashboards)
```

Comment:
https://github.com/sourcegraph/deploy-sourcegraph-dot-com/tree/feature/release/dashboards is 404 @pecigonzalo 😁

Reply:
I think this link has since been fixed! https://github.com/sourcegraph/deploy-sourcegraph-dot-com/tree/release/dashboards

Reply:
@bobheadxi thanks for checking that!

On this README though, something odd going on: https://github.com/sourcegraph/deploy-sourcegraph-dot-com/blob/release/dashboards/README.md

![CleanShot 2021-01-18 at 18 04 48@2x](https://user-images.githubusercontent.com/2946214/104901088-de654c80-59b7-11eb-9787-3634216535e6.png)

Reply:
https://github.com/sourcegraph/deploy-sourcegraph-dot-com/pull/4196 !